spec/features/cases/overturned_sar/create_case_spec.rb

```diff
@@ -45,6 +45,50 @@
                       type: :text
                     )
 
+    expect(form).not_to have_flag_for_disclosure_specialists
+
+    click_button 'Create case'
+
+    # Browse Business Group
+    assignments_new_page.choose_business_group(responding_team.business_group)
+
+    # Select Business Unit
+    assignments_new_page.choose_business_unit(responding_team)
+  end
+
+  scenario 'creating an OVT FOI case', js: true do
+    ico_case = create(:closed_ico_foi_case)
+
+    # Move this to a user journey test once displaying works.
+    # create_and_assign_overturned_sar_case(user: manager,
+    #                                       ico_case: ico_case,
+    #                                       responding_team: responding_team)
+
+    cases_show_page.load(id: ico_case.id)
+    expect(cases_show_page).to be_displayed(id: ico_case.id)
+    # Replace the following-line with a click on the "New overturned case"
+    # button when available
```

Comment:
I'm pretty sure we have a user journey that tests that the button comes up on the ICO page, etc, so we can probably get rid of the bits that load the ICO page first and related comments.